parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -291,6 +293,8 @@ try
     icacls.exe "c:\k"
     Get-ProvisioningScripts
     Get-LogCollectionScripts
+
+    Disable-KubeletServingCertificateRotationForTags

```

Comment:
this is just for testing, @AbelHu we'll need to remove/comment this out until the script package changes roll out right?